api/src/main/resources/liquibase.xml

```diff
@@ -195,4 +198,16 @@
 			DELETE FROM person_attribute WHERE value IS NULL OR value = '';
 		</sql>
 	</changeSet>
+	<changeSet id="ugandaemr-01-10-2018-1200" author="slubwama">
+        <preConditions onFail="CONTINUE">
```

Comment:
@slubwama The expected result for your SQL check is 1 not 0, so that the changeset is run only when the specified program exists

Reply:
@ssmusoke that was handled before this comment